runners/flink/runner/src/main/java/org/apache/beam/runners/flink/translation/FlinkStreamingTransformTranslators.java

```diff
@@ -189,7 +189,11 @@ public void translateNode(Read.Bounded<T> transform, FlinkStreamingTranslationCo
           new FlatMapFunction<T, WindowedValue<T>>() {
             @Override
             public void flatMap(T value, Collector<WindowedValue<T>> out) throws Exception {
-              out.collect(WindowedValue.of(value, Instant.now(), GlobalWindow.INSTANCE, PaneInfo.NO_FIRING));
+              out.collect(
+                  WindowedValue.of(value,
+                    BoundedWindow.TIMESTAMP_MIN_VALUE,
+                    GlobalWindow.INSTANCE,
+                    PaneInfo.NO_FIRING));
             }
           }).assignTimestampsAndWatermarks(new IngestionTimeExtractor<WindowedValue<T>>());
```

Comment:
I know nothing here, just want to confirm that it's okay to use an "IngestionTimeExtractor" for a collection where all elements have timestamps of MIN_VALUE.